isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -4350,26 +4316,26 @@ namespace isobus
 			return false;
 		}
 
-		if (replace && replace_command(data, priority))
+		if (replace && replace_command(data))
 		{
 			return true;
 		}
 
-		commandQueue.emplace_back(data, priority);
+		commandQueue.emplace_back(data);
 		return true;
 	}
 
-	bool VirtualTerminalClient::replace_command(const std::vector<std::uint8_t> &data, CANIdentifier::CANPriority priority)
+	bool VirtualTerminalClient::replace_command(const std::vector<std::uint8_t> &data)
 	{
 		bool alreadyReplaced = false;
 		for (auto it = commandQueue.begin(); it != commandQueue.end();)
 		{
-			bool matchesFunctionCode = (it->first[0] == data[0]);
-			if (matchesFunctionCode && (it->second == priority))
+			bool matchesFunctionCode = (it[0][0] == data[0]);
+			if (matchesFunctionCode)
```

Comment:
```suggestion
			bool matchesFunctionCode = (it[0] == data[0]);
			if (matchesFunctionCode)
```
I think this is better? Not sure if it makes a difference

Reply:
Or maybe *it[0]?

Reply:
Nah, this doesn't work. I've tried and the only thing that seems to work is it[0][0] since it is a vector of a vector and data is just a vector.

Reply:
Oh yeah, I can totally see why that's confusing. Let me explain; basically a `std::vector<X>::iterator` means it a iterator/pointer to an element of it's suffix. So essentially the whole `std::vector<X>::iterator` points to an element X (which lies inside the vector), and hence in our case, it point to an inner `std::vector<uint8_t>` element. Does that help?